modules/compliance-anatomy.adoc

```diff
@@ -0,0 +1,276 @@
+----
+Events:
+  Type     Reason        Age    From                    Message
+  ----     ------        ----   ----                    -------
+  Normal   SuiteCreated  9m52s  scansettingbindingctrl  ComplianceSuite openshift-compliance/my-companys-compliance-requirements created
+----
+
+Now a `ComplianceSuite` object is created. The flow continues to reconcile the newly created `ComplianceSuite`.
+
+== ComplianceSuite lifecycle and debugging
+The `ComplianceSuite` CR is a wrapper around `ComplianceScan` CRs. The `ComplianceSuite` CR is handled by controller tagged with `logger=suitectrl`.
+This controller handles creating Scans from a Suite, reconciling and aggregating individual Scan statuses into a single Suite status. If a Suite is set to execute periodically, the `suitectrl` also handles creating a `CronJob` CR that re-runs the Scans in the Suite after the initial run is done:
+
+[source,terminal]
+----
+oc get cronjobs
+NAME                                           SCHEDULE    SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+my-companys-compliance-requirements-rerunner   0 1 * * *   False     0        <none>          151m
+----
+
+For the most important issues, Events are emitted. View them with `oc describe compliancesuites/$name`. The Suite objects also have a Status subresource that is updated when any of Scan objects that belong to this suite update their Status subresource. After all expected scans are created, control is passed to the scan controller.
+
+== ComplianceScan lifecycle and debugging
+The `ComplianceScan` CRs are handled by the `scanctrl` controller. This is also where the actual scans happen and the scan results are created. Each scan goes through several phases:
+
+=== Pending phase
+The scan is validated for correctness in this phase. If some parameters like storage size are invalid, the scan transitions to DONE with ERROR result, otherwise proceeds to the Launching phase.
+
+=== Launching phase
+In this phase, several `ConfigMaps` that contain either environment for the scanner pods or directly the script that the scanner pods will be evaluating. List the ConfigMaps:
+
+[source,terminal]
+----
+oc get cm -lcompliance.openshift.io/scan-name=rhcos4-e8-worker,complianceoperator.openshift.io/scan-script=
+----
+
+These `ConfigMaps` will be used by the scanner pods. If you ever needed to modify the scanner behavior, change the scanner debug level or print the raw results, modifying the `ConfigMaps` is the way to go. Afterwards, a `PersistentVolumeClaim` is created per scan in order to store the raw ARF results:
++
+[source,terminal]
+----
+oc get pvc -l$scan_name
+----
+
+The PVCs are mounted by a per-scan `ResultServer` deployment. A `ResultServer` is a simple HTTP server where the individual scanner pods, each of which might be running on a different node, upload the full ARF results to. This might seem odd, but the full ARF results might be quite big and at the same time, without knowing details about the storage
```

Comment:
A `ResultServer` is a simple HTTP server where the individual scanner pods upload the full ARF results to. Each of server can run on a different node.

I would delete "This might seem odd, but" and perhaps break up the lengthy sentences here.